.cirrus.yml

```diff
@@ -154,10 +136,6 @@ task:
     image: big-sur-xcode-12.3
   upgrade_script:
     - sudo gem install cocoapods
-    - flutter channel stable
-    - flutter upgrade
-    - flutter channel beta
-    - flutter upgrade
```

Comment:
We can all remove all the `flutter channel $CHANNEL` and `flutter channel master` calls in the tasks, right? Doesn't really matter, but I don't know how no-op-y that is in the tool so it might save a bit of work.